data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1616 @@
+			`	"We have made good progress since you were here last. We have managed to take over control of life support systems in several wings of rooms, and now those are run entirely from our own systems. Aside from relieving a tiny amount of load from the station's power core, it also means that the station could fail completely and we would still have safe living quarters to serve as base of operations."`
+			`	He pauses, and is gestures take on a tone of disgust. "Of course, we are working to avoid that. But having multiple backup systems is the key to survival in hostile conditions."`
+			`	He looks around at the few Remnant who have yet to vanish into the rest of the station. "Thank you for bringing more explorers." He hands you a datacrystal. "I have already transmitted this data to Viminal, but I would appreciate it if you could take this to them to ensure there are no errors or data corruption in the transmission.`
+
+
+
+mission "Remnant: Cognizance 30"
+	name "Deliver datacrystal to Viminal"
+	description "Gavriil has asked you to take a datacrystal to Viminal."
+	landing
+	source "Ssil Vida"
+	destination "Viminal"
+	to offer
+		has "Remnant: Cognizance 29: done"
+	on complete
+		conversation
+			`Not knowing who to deliver the data crystal to, you take it to the information desk. The Remnant currently working there gestures that they have been expecting it. They thank you for delivering it. As you leave the information desk, your commlink chimes with a message from prefect Chilia. It informs you that Chilia would like to speak with you, and is currently in the spaceport cafeteria.`
+
+
+
+mission "Remnant: Cognizance 31"
+	name "Pickup Refined Fuel"
+	description "Chilia has asked you to pick up a refined fuel from Clink by <date>."
+	deadline 19
+	cargo "Reserved" 58
+	illegal -1 "Your communication panel beeps insistently with a message that a scan has been detected, and thus the mission has been terminated."
+	stealth
+	infiltrating
+	source "Viminal"
+	destination "Clink"
+	to offer
+		has "Remnant: Cognizance 30: done"
+		has "license: Remnant Capital"
+	on offer
+		conversation
+			`Chilia is just collecting his tray from the counter when you step into the cafeteria. Seeing you enter, he picks up a second and nods towards a pair of open seats at a table with Taely and Dusk. "We are glad you could make it, <first>." Taely looks happy, brimming with contained energy. "It is an exciting day, Captain, but Dusk should lead." She nods to Dusk, who sets down his fork to gesture. "While Gavriil has been making excellent progress setting up facilities on Ssil Vida, I hae been working with a team of xenolinguists on deciphering the data we have collected so far. Based on Gavriil's recommendations, we have been focusing on the power distribution system.`
```

Comment:
```suggestion
			`Chilia is just collecting his tray from the counter when you step into the cafeteria. Seeing you enter, he picks up a second and nods towards a pair of open seats at a table with Taely and Dusk. "We are glad you could make it, <first>." Taely looks happy, brimming with contained energy. "It is an exciting day, Captain, but Dusk should lead." She nods to Dusk, who sets down his fork to gesture. "While Gavriil has been making excellent progress setting up facilities on Ssil Vida, I have been working with a team of xenolinguists on deciphering the data we have collected so far. Based on Gavriil's recommendations, we have been focusing on the power distribution system.`
```

Reply:
I'm not sure why I can't just commit your suggestion, but in any case, thanks for spotting it. I've made the correction manually based on your suggestion.

Thanks!